### PR TITLE
Fixups for UEFI_SECURITY.md

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

**Comment by keszybz** [comment]
LGTM.

File: src/boot/efi/UEFI_SECURITY.md [hunk 1]
```diff
@@ -24,13 +24,15 @@ Specification)](https://uapi-group.org/specifications/specs/boot_loader_specific
...
-
-Since it is embedded in a PE signed binary, `systemd-stub` will temporarily disable SecureBoot
-authentication when loading the payload kernel it wraps, in order to avoid redundant duplicate
-authentication of the image, given that that the payload kernel was already authenticated and verified as
-part of the whole image. SecureBoot authentication is re-enabled immediately after the kernel image has been
-loaded.
+additional content such as configuration or optional services. [Unified Kernel
+Images](https://uapi-group.org/specifications/specs/unified_kernel_image/) embed `systemd-stub`, a kernel
+and other optional components as sections in a PE signed binary, that can thus be executed in UEFI
+environments.
```
Comment (poettering) [inline src/boot/efi/UEFI_SECURITY.md:30]: as mentioned before, this is not what makes them executable in UEFI mode, the kernel upstream has a stub anyway, which we jump into... this now makes it sound as if our stub was necessary to make the kernel executable as an uefi program, but that's not the case.

File: src/boot/efi/UEFI_SECURITY.md [hunk 2]
```diff
@@ -24,13 +24,15 @@ Specification)](https://uapi-group.org/specifications/specs/boot_loader_specific
...
+
+Since it is embedded in a PE signed binary, `systemd-stub` will temporarily disable the UEFI authentication
+protobol while loading the payload kernel it wraps, in order to avoid redundant duplicate authentication of
```
Comment (poettering) [inline src/boot/efi/UEFI_SECURITY.md:33]: "protobol" is a funny word

Comment (keszybz) [reply src/boot/efi/UEFI_SECURITY.md:33]: I think it's an accurate way to describe the UEFI mess.

**Pull request merged**